From 5934fa0a0708b551211f63b8d91676df346e9821 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <vladimir.oltean@nxp.com>
Date: Fri, 29 Nov 2019 03:13:00 +0200
Subject: [PATCH] arm64: dts: fsl: ls1028a: Specify that the Felix port 4 runs
 at 2.5Gbps

This is just an informative change, because all Felix MACs inside the
LS1028A are hardwired in gigabit mode anyway.

Only PHYLINK is able to understand fixed-link speeds higher than 1 Gbps.
With PHYLIB, fixed-link interfaces are emulated as C22 PHYs by the swphy
driver, and C22 does not specify settings for speeds higher than
gigabit.

This patch brings no functional change except for the messages printed
during driver initialization.

Signed-off-by: Vladimir Oltean <vladimir.oltean@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -815,7 +815,7 @@
 						phy-mode = "gmii";
 
 						fixed-link {
-							speed = <1000>;
+							speed = <2500>;
 							full-duplex;
 						};
 					};
